slave/buildslave/test/unit/test_commands_bk.py

```diff
@@ -63,7 +65,7 @@ def test_simple(self):
         ]
         self.patch_runprocess(*expects)
 
-        d = self.run_command()
+        yield self.run_command()
+
         # TODO: why the extra quotes?
-        d.addCallback(self.check_sourcedata, '"http://bkdemo.bkbits.net/bk_demo1\n"')
-        return d
+        self.assertSourceData('"http://bkdemo.bkbits.net/bk_demo1\n"')
```

Comment:
While you are at it, could you please fix the domain name to something `example`?  E.g. `bkdemo.example.net`?